.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,9 +1,7 @@
 ---
-name: "✨ Feature request"
-about: Suggest a new feature or idea
-title: ''
-labels: "feature request, awaiting triage"
-assignees: ''
+name: "Feature request"
+about: Suggest a new feature or idea in libraries, services, and tools
+labels: "accepted task, accepted spike, cantfix, triage, wontfix"
```

Comment:
@JoPintoPaul based on the old value, are these the default labels that a ticket using this template gets created with? Where it seems like these are all mutually exclusive status labels

Reply:
@oscarduignan You are quite right, I had misunderstood this syntax. Updated to remove default labels entirely.